src/Tabs/Indicator.tsx

```diff
@@ -10,33 +10,42 @@ interface Props {
 
 const Indicator: FunctionComponent<Props> = ({ activeTabElement, position, duration }) => {
   const [style, setStyle] = useState<any>({})
-
+  const [width, setWidth] = React.useState(window.innerWidth);
```

Comment:
yeah, this will break on the server side. Either setWidth on a `useEffect` mount or conditionally check the window with the `isClient` util